packages/site-middleware/src/plugins/prettyCode.ts

```diff
@@ -0,0 +1,42 @@
+import { getHighlighter } from 'shiki';
+
+const options = {
+  getHighlighter: highlighterOptions =>
+    getHighlighter({
+      ...highlighterOptions,
+      langs: ['bat', 'diff', 'css', 'json', 'jsx', 'tsx', 'mdx', 'yaml', 'ts']
+    }),
+  theme: {
+    dark: 'dark-plus',
+    light: 'light-plus'
```

Comment:
I don't love how the light theme looks, is it easy to switch it out?

Reply:
Very easy to change. I'm not a huge fan either but picked this with a11y in mind.